lib/services/profiles-api-service.js

```diff
@@ -213,6 +215,14 @@ function profileApiServiceFactory(
             });
         })
         .then(function () {
+            if (options.type === 'switch') {
+                return waterline.nodes.updateByIdentifier(
+                    node.id,
+                    { snmpSettings: { host: ip, community: community} }
```

Comment:
It's true that there would be more flexilibity in exposing as a task.  I did think it was pretty safe however since likely we will control startup-config settings and it will probably be deployed only through the automated discovery process. Do you think users will create and deploy their own startup-config files?  The nice part of doing it this way, in my mind, is it is all automated.